crates/ruff/src/settings/types.rs

```diff
@@ -92,6 +92,23 @@ impl PythonVersion {
     }
 }
 
+#[derive(Clone, Copy, Debug, PartialOrd, Ord, PartialEq, Eq, Default, CacheKey, is_macro::Is)]
```

Comment:
```suggestion
#[derive(Clone, Copy, Debug, PartialEq, Eq, Default, CacheKey, is_macro::Is)]
```

I think we can remove the ordering from PreviewMode. It's unclear to me where this would be useful (and what the expected ordering should be)

Reply:
👍